.travis.yml

```diff
@@ -1,52 +1,50 @@
 sudo: required
 dist: trusty
 language: node_js
-
 node_js:
 - '6.9.1'
 - 'stable'
-
 addons:
   apt:
     sources:
-    - ubuntu-toolchain-r-test
     - google-chrome
+    - ubuntu-toolchain-r-test
     packages:
     - google-chrome-stable
     - g++-4.8
   firefox: 'latest-esr'
-
+cache:
+  directories:
+    - node_modules
 env:
   matrix:
-  - EMBER_TRY_SCENARIO=ember-1-13
+  - EMBER_TRY_SCENARIO=ember-2-3
```

Comment:
The README says we support 1.13.0+ so we should probably keep testing it or alter the README to say we support 2.3+.